libs/community/langchain_community/llms/anyscale.py

```diff
@@ -24,10 +24,7 @@
 from langchain_community.utils.openai import is_openai_v1
 
 DEFAULT_BASE_URL = "https://api.endpoints.anyscale.com/v1"
-DEFAULT_MODEL = "Meta-Llama/Llama-Guard-7b"
-
-# Completion models support by Anyscale Endpoints
-COMPLETION_MODELS = ["Meta-Llama/Llama-Guard-7b"]
+DEFAULT_MODEL = "mistralai/Mixtral-8x7B-Instruct-v0.1"
```

Comment:
changing default model is a breaking change, would it be better to add a warning that llama guard is deprecated?

Reply:
Changing the model is not really affect the output, still will generate LLM responses for the prompt. 
 From Anyscale backend, we can see the previous Llama guide model has very low traffic, so the change won't be breaking to users ( That's also the reason we deprecate this model from public hosting)